inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_matmul.cpp

```diff
@@ -0,0 +1,138 @@
+    }
+};
+
+class DSR_MatMulAInput : public testing::WithParamInterface<Parameters>,
+        public LayerTestsUtils::LayerTestsCommon {
+protected:
+    void SetUp() override {
+        const auto& parameters = GetParam();
+        const auto& data_type = std::get<0>(parameters);
+        const auto& matmul_setup = std::get<1>(parameters);
+        targetDevice = std::get<2>(parameters);
+
+        const auto input_A = std::make_shared<ngraph::opset3::Parameter>(data_type, matmul_setup.A.shape);
+        const auto input_B = std::make_shared<ngraph::opset3::Parameter>(data_type, matmul_setup.B.shape);
+
+        const auto input_A_dsr = std::make_shared<ngraph::opset3::Parameter>(ngraph::element::i64, ngraph::Shape{matmul_setup.A.shape.size()});
+        const auto dsr_A = std::make_shared<ngraph::vpu::op::DynamicShapeResolver>(input_A, input_A_dsr);
+
+        const auto node = std::make_shared<ngraph::opset3::MatMul>(dsr_A, input_B, matmul_setup.A.transpose, matmul_setup.B.transpose);
+
+        const auto result = std::make_shared<ngraph::opset3::Result>(node);
+        function = std::make_shared<ngraph::Function>(ngraph::ResultVector{result},
+                ngraph::ParameterVector{input_A, input_B, input_A_dsr}, "DSR-MatMul");
+    }
+};
+
+class DSR_MatMulBInput : public testing::WithParamInterface<Parameters>,
+        public LayerTestsUtils::LayerTestsCommon {
+protected:
+    void SetUp() override {
+        const auto& parameters = GetParam();
+        const auto& data_type = std::get<0>(parameters);
+        const auto& matmul_setup = std::get<1>(parameters);
+        targetDevice = std::get<2>(parameters);
+
+        const auto input_A = std::make_shared<ngraph::opset3::Parameter>(data_type, matmul_setup.A.shape);
+        const auto input_B = std::make_shared<ngraph::opset3::Parameter>(data_type, matmul_setup.B.shape);
+
+        const auto input_B_dsr = std::make_shared<ngraph::opset3::Parameter>(ngraph::element::i64, ngraph::Shape{matmul_setup.B.shape.size()});
+
+        const auto dsr_B = std::make_shared<ngraph::vpu::op::DynamicShapeResolver>(input_B, input_B_dsr);
+
+        const auto node = std::make_shared<ngraph::opset3::MatMul>(input_A, dsr_B, matmul_setup.A.transpose, matmul_setup.B.transpose);
+
+        const auto result = std::make_shared<ngraph::opset3::Result>(node);
+        function = std::make_shared<ngraph::Function>(ngraph::ResultVector{result},
+                ngraph::ParameterVector{input_A, input_B, input_B_dsr}, "DSR-MatMul");
+    }
+};
+
+TEST_P(DSR_MatMul, CompareWithReference) {
+    Run();
+}
+INSTANTIATE_TEST_CASE_P(DISABLED_DynamicMatMul, DSR_MatMul, combinations);
+
+TEST_P(DSR_MatMulAInput, CompareWithReference) {
+    Run();
+}
+INSTANTIATE_TEST_CASE_P(DISABLED_DynamicMatMul, DSR_MatMulAInput, combinations);
+
+TEST_P(DSR_MatMulBInput, CompareWithReference) {
+    Run();
+}
+INSTANTIATE_TEST_CASE_P(DISABLED_DynamicMatMul, DSR_MatMulBInput, combinations);
```

Comment:
Why tests are disabled?

Reply:
Enabled them for now. But I have no idea what to expect -- I have no access to my stick for now. Could you please help with error identification in CI, if any?